experiments/warmup0/setup

```diff
@@ -1,28 +1,44 @@
 #!/bin/bash
 
-. environment.conf
-cd execute
-
-git clone $promethee_url
-cd Promethee2
-git checkout caja
-git reset --hard $promethee_version
-
-cd ..
-git clone $tiff_utils_url
-cd TiffUtils
-git reset --hard $tiff_utils_version
-g++ -std=c++14 tifGenerator.cpp -o tifGenerator -ltiff
-./tifGenerator 4000 tifrandom.tif -R
-rm tifGenerator
-
-cd ..
-g++ writeCriteriaInfo.cpp -o writeCriteriaInfo
-./writeCriteriaInfo > Promethee2/promethee/caja/playground/src/criteria
-rm writeCriteriaInfo
-
-mv TiffUtils/tifrandom.tif Promethee2/promethee/caja/playground/src
-cd Promethee2/promethee/caja/playground/src/
+source environment.conf
+
+rm -rf $results_path
+
+mkdir code
+mkdir results
+
+if [ ! -d $code_path/$promethee_path ]; then
+	git clone $promethee_url
+	mv $promethee_path $code_path
+
+	cd $code_path/$promethee_path
+	git checkout --detach $promethee_version
+	cd -
+fi
+
+if [ ! -d $code_path/$tiff_utils_path ]; then
+	git clone $tiff_utils_url
+	mv $tiff_utils_path $code_path
+
```

Comment:
avoid `cd`. Why not check the code out in the current directory then `mv` it after